### ISSUE TITLE
Keyword matching isn't inclusive enough?

### ISSUE THREAD

**Issue by SevereOverfl0w** [clj-commons/rewrite-cljs#6]
https://github.com/snoe/clj-refactor.nvim/issues/2 issue coming from here.

rewrite-cljs is unable to parse the following correctly:

`(->> {:select [:%count.*]`

error is: [Error: Invalid token: :, on line: 485, on column: 19] Error: Invalid token: :, on line: 485, on column: 19

**Comment by rundis** [comment]
I'm pretty sure this is fixed in 0.4.0.

**Comment by SevereOverfl0w** [comment]
ah, is the API compatible?

**Comment by rundis** [comment]
Shouldn't be any api changes. The major thing is upgrade to cljs 1.7 and using tools.reader rather than bespoke reader for most read operations when parsing.

**Comment by SevereOverfl0w** [comment]
Yeah, looks like a good change. Making a push upstream to fix this in the vim plugin, and I'll double check it's gone.

**Comment by rundis** [comment]
Cool. Wasn't aware of that vim-plugin :smile:

**Comment by snoe** [comment]
Hi rundis, thanks for the great port.

Confirming that this is solved with 0.4.0. However, the dependency tree is a bit confused: 

```
Possibly confusing dependencies found:
[org.clojure/clojurescript "1.7.228"] -> [org.clojure/tools.reader "1.0.0-alpha1"]
 overrides
[rewrite-cljs "0.4.0"] -> [org.clojure/tools.reader "1.0.0-alpha3"]

Consider using these exclusions:
[rewrite-cljs "0.4.0" :exclusions [org.clojure/tools.reader]]
```

I followed the suggestion but it seems tools.reader [alpha1 and alpha2 are not compatible](https://github.com/clojure/tools.reader/blob/master/CHANGELOG.md) (and there's no changelog for alpha3). Some libraries like om and [core.async use `:scope "provided"`](https://github.com/clojure/core.async/blob/master/project.clj#L9) for clojurescript so that might be a possibility in the project, otherwise I think you want to depend on the same tools.reader version as the one brought along by the version of cljs you are depending on.

**Comment by rundis** [comment]
I'm closing this. Tools reader has been upped to a released version, hopefully compatible with most commonly used cljs versions in use these days.
